packages/components/src/Loading/LoadingSpinner/LoadingSpinner.module.css

```diff
@@ -0,0 +1,28 @@
+.loadingSpinner {
+  --loading-spinner-size: var(--spacing-48);
```

Comment:
If we expect that size will always exist, we could probably safely move this into a class so it's clear that it's for `md`. 
That being said, I could see the argument for missing a default I guess 🤷🏻‍♀️  I'll let you decide :)

Reply:
My thinking is to declare the CSS var here as the default since I remember coming into some issue leaving one undefined on the base selector. Seemed a little hat on a hat to re-declare it in a .md class but could see that making sense for clarity